Changelog

```diff
@@ -2,9 +2,12 @@ Next release:
   Major features:
     * Vk.com support
     * Seafile support
+    * Amazon Cloud Drive support
     * Add Ampache compatibility (through Subsonic service)
     * Add new analyzer "Rainbow Dash"
     * Answer to the ultimate question of life, the universe and everything
+    * Add "Psychedelic Colour" mode to all analyzers
+    * Add mono output support in the engine
```

Comment:
This was already there and just temporarily removed in the gstreamer 1.0 transition IIRC